Breeze/src/Breeze.Wallet/Controllers/WalletController.cs

```diff
@@ -131,9 +130,53 @@ public IActionResult GetInfo([FromQuery] WalletName model)
 			}
 		}
 
+		[Route("sensitive")]
```

Comment:
decided not to include this at this point as I'm not sure it's needed by the front-end.